client/java/src/main/java/com/linecorp/centraldogma/client/AbstractCentralDogmaBuilder.java

```diff
@@ -415,4 +419,16 @@ public final B retryIntervalOnReplicationLagMillis(long retryIntervalOnReplicati
     protected long retryIntervalOnReplicationLagMillis() {
         return retryIntervalOnReplicationLagMillis;
     }
+
+    /**
+     * Sets the {@link MeterRegistry} used to collect metrics.
+     */
+    public final B meterRegistry(MeterRegistry meterRegistry) {
+        this.meterRegistry = Optional.of(requireNonNull(meterRegistry, "meterRegistry"));
+        return self();
+    }
+
+    protected Optional<MeterRegistry> meterRegistry() {
```

Comment:
Could we use `@Nullable` instead of `Optional` for better interop with other JVM languages such as Kotlin that translate `@Nullable` as `?` type?
https://kotlinlang.org/docs/java-interop.html#nullability-annotations